src/pymor/algorithms/riccati.py

```diff
@@ -123,6 +128,15 @@ def solve_ricc_lrcf(A, E, B, C, R=None, S=None, trans=False, options=None,
         from pymor.bindings.slycot import solve_ricc_lrcf as solve_ricc_impl
     elif backend == 'pymess':
         from pymor.bindings.pymess import solve_ricc_lrcf as solve_ricc_impl
+    elif backend == 'lrradi':
+        if R is None and S is None:
+            from pymor.algorithms.lrradi import solve_ricc_lrcf as solve_ricc_impl
+            return solve_ricc_impl(A, E, B, C, trans=trans, options=options)
+        elif config.HAVE_PYMESS:
+            logger.log(f'Low-rank RADI does not support inputs R and S, switching to pymess')
+            from pymor.bindings.pymess import solve_ricc_lrcf as solve_ricc_impl
+        else:
+            raise ValueError(f'Low-rank RADI does not support inputs R and S.')
```

Comment:
I would prefer for the `elif backend == 'lradi'` to look like `elif backed == 'pymess'` and for `lrradi.solve_ricc_lrcf` to raise `NotImplementedError` like `bindings.pymess.solve_ricc_lrcf`.